manifests/operator-rbac.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: zalando-postgres-operator
+
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: zalando-postgres-operator
+rules:
+- apiGroups:
+  - acid.zalan.do
+  resources:
+  - postgresqls
+  verbs:
+  - "*"
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
```

Comment:
This did not work as we are defining a `Role` whereas the operation being performed is at cluster level, so there should be another separate `ClusterRole` and corresponding `ClusterRoleBinding` and actions being performed at cluster level must be separated out.

Reply:
I guess everything will be at the cluster level assuming the operator is running with multi-namespace support.

Reply:
we don't have support for multiple namespaces at the moment, but this patch https://github.com/zalando-incubator/postgres-operator/pull/225 aims to add support for listening to manifests and creating clusters either in a single namespace (as it is today), or in all namespaces, therefore, defining a `ClusterRole` makes sense to me. It will also solve the issue of creating customresourcedefinitions.

Reply:
apparently, another argument for the cluster role is that the operator needs to list nodes in order to migrate from the nodes approaching EOL (as indicated by the lack of the `node_readiness_label`).

Reply:
> patch #225 aims to add support for listening to manifests and creating clusters either in a single namespace (as it is today), or in all namespaces,

it is now merged, so we do have support for multiple namespaces and thus should opt for `ClusterRole`